libass/ass_rasterizer.h

```diff
@@ -26,6 +26,8 @@
 #include "ass_bitmap.h"
 
 
+#define RASTERIZER_PRECISION 16  // rasterizer spline approximation error in 1/64 pixel units
```

Comment:
Oh, I'd forgotten I can just pass `RenderContext` in the `priv` argument to `construct` instead of `ASS_Renderer`. Sure, then I can get rid of the thread-local stuff altogether.